appveyor.yml

```diff
@@ -0,0 +1,33 @@
+version: '{build}'
+
+image: Visual Studio 2017
+
+pull_requests:
+  do_not_increment_build_number: true
+
+branches:
+  only:
+    - dev
+    - master
+
+environment:
+  matrix:
+    - JAVA_HOME: C:\Program Files\Java\jdk1.8.0
+
+install:
+  - cmd: echo %JAVA_HOME%
+  - ps: Get-Command mvn
+  - cmd: mvn -v
+  - ps: Get-Command nuget
+
+build_script:
+  - mvn clean install -DskipTests
+  - ps: nuget pack ./tools/AzureFunctionsJavaWorker.nuspec -Properties version=$env:APPVEYOR_BUILD_NUMBER
```

Comment:
Would recommend renaming the property to `versionsuffix` (that's what we use in the other projects) or something the better describes this, as the actual version is the concatenation of major.minor and this property

Reply:
Should I make version a modifiable field as well?

Reply:
I could just make the whole thing $version$ and assemble the string via a PS expression, right?

Reply:
Up to you. Usually, the major minor go in the nuspec and we just take the suffix as a parameter.